force-app/main/default/lwc/autoEnrollmentMappingModalBody/autoEnrollmentMappingModalBody.js

```diff
@@ -90,6 +84,7 @@ export default class autoEnrollmentMappingModalBody extends LightningElement {
     }
 
     handleAccountRecordTypeChange(event) {
+        this.accountRecordTypeComboboxVModel.value = event.detail.value;
```

Comment:
I was just wondering if we need this line

Reply:
I had added it as the html displays the API name of the selected record type through `accountRecordTypeComboboxVModel.value`, but this value isn't set anymore when when using imperative apex as that only runs once and before user interaction.

Before, with the wire, the LWC dispatches the event that the value has changed and then the aura wrapper would set `newAccountRecordType` to the selected combobox value which would cause the wire in LWC to re-run as it's passed as a parameter (and the return data appropriately sets the selected value in `accountRecordTypeComboboxVModel.value`). 

With imperative, the aura wrapper changing the value of `newAccountRecordType` doesn't cause anything to happen (with how I did it) so I had added this line to capture the value before dispatching the event. It technically goes against the framework - I had initially thought of using a getter/setter for `newAccountRecordType` so that I could have more control to call the imperative apex again when the selection changes to match what the framework does with wires? It's more calls to the server and more code so I had ended up going with just this one line. Open to suggestions!

Reply:
@kg345 you're totally right, no need to change anything :-). Thanks for explaining this.